projects/plugins/mu-wpcom-plugin/mu-wpcom-plugin.php

```diff
@@ -3,7 +3,7 @@
  *
  * Plugin Name: WordPress.com Features
  * Description: Test plugin for the jetpack-mu-wpcom package
- * Version: 2.0.8
+ * Version: 2.0.9
```

Comment:
You mention this PR backports for mu-wpcom 2.0.10, is this 2.0.9 ok? Same on package.json below.

Reply:
Good catch, this was probably a typo in the script that I ran locally. The version in the code is the one going to be used.

Reply:
I have edited the PR description and header to avoid confusion, thanks!